snapshot/git/gitdb/db_test.go

```diff
@@ -130,6 +130,112 @@ func TestStream(t *testing.T) {
 
 }
 
+func TestStreamInit(t *testing.T) {
+	// This test doesn't use our fixture DBs because it has such specific git setup
+	// Our git repos are:
+	// rw: the main, upstream read-write repo
+	// mirror: where to init from (actually not a repo; just a bundle)
+	// ro: a read-only version of rw
+	// data: the data repo for our db
+
+	// our initer will fetch from the bundle "mirror"
+
+	// our test plan is:
```

Comment:
Well, the flip side is, it's harder to get a sense of it beforehand... but you're probably right. Done.